python/perf-kernels/rmsnorm.py

```diff
@@ -16,6 +16,14 @@ def is_hip():
     return triton.runtime.driver.active.get_current_target().backend == "hip"
 
 
+if torch.cuda.is_available():
+    current_device_index = torch.cuda.current_device()
+    current_device = torch.cuda.get_device_properties(current_device_index)
+    NUM_SMS = current_device.multi_processor_count
+else:
+    NUM_SMS = 304
+
+
 def get_cuda_autotune_config():
```

Comment:
Can we remove this? We don't want to maintain nv side.

Reply:
yes, the aim is to automatically detect num_sms. see above comments